src/Behat/Behat/Output/Node/EventListener/AST/OutlineTableListener.php

```diff
@@ -215,7 +216,9 @@ private function printExampleRowOnAfterExampleEvent(Formatter $formatter, Event
             $this->stepSetupPrinter->printSetup($formatter, $beforeEvent->getSetup());
         }
 
-        $this->exampleRowPrinter->printExampleRow($formatter, $this->outline, $example, $this->stepAfterTestedEvents);
+        if ($example instanceof ExampleNode) {
+            $this->exampleRowPrinter->printExampleRow($formatter, $this->outline, $example, $this->stepAfterTestedEvents);
+        }
```

Comment:
Same observation as the other Outline-related listener above, I think perhaps we should throw?

Reply:
I've updated the code to use assertions, which is what PHPStan recommends for this kind of type narrowing. What do you think?

Reply:
That looks like a good solution to me